pkg/config/config.go

```diff
@@ -243,6 +244,7 @@ func setDefaults(v *viper.Viper) {
 	v.SetDefault("options.template_event_topic", "platform.content-sources.template")
 	v.SetDefault("options.repository_import_filter", "")
 	v.SetDefault("options.external_url", "http://pulp.content:8000")
+	v.SetDefault("options.snapshot_retain_days_limit", 90)
```

Comment:
i'd probably set the default to a higher value (365), and then we can customize it in app-interface to do less for stage